regulations/static/regulations/js/source/resources.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
I'll add some comments. The file is to pull out the pieces that need to be mocked in tests. 
I named it resources.js so more objects can be added for other functions that need to be refactored, but I'm not opposed to changing the name or doing this in a different way.